docs/modules/ROOT/pages/documentation/payara-server/public-api/openid-connect-support.adoc

```diff
@@ -229,16 +226,16 @@ They are detailed below.
 | @LogoutDefinition
 | Defines the functionality that is performed when the user logs out and defines the RP Session Management configuration.
 |
-| 
+|
 |===
 
-*Note :* If both an annotation attribute and a MicroProfile Config property are defined for the same option 
+NOTE: If both an annotation attribute and a MicroProfile Config property are defined for the same option
 then the MicroProfile Config property value always takes precedence over the `@OpenIdAuthenticationDefinition` annotation value.
 
```

Comment:
Move the table about `@OpenIdProviderMetadata` options to the main configuration section and add configuration about `endSessionEndpoint`.

```suggestion

.Configuration Options For `@OpenIdProviderMetadata`
|===
| Option | MP Config property | Description | Default value

| `endSessionEndpoint`
| `payara.security.openid.provider.endSessionEndpoint`
| OIDC provider’s logout endpoint URL. If set, overrides the URL obtained via the `end_session_endpoint` element of the OIDC provider’s metadata.
|  

| `disableScopeValidation`
| `payara.security.openid.disableScopeValidation`
| This property disables the scope validation for custom scope configurations
| false 

```